README.md

```diff
@@ -812,6 +817,81 @@ config.serialization_config.data_serializable_factories[1] = factory
 
 Note that the ID that is passed to the `SerializationConfig` is same as the factory ID that `Foo` class returns.
 
+### 4.2.1. Versioning for Portable Serialization
+
+More than one version of the same class may need to be serialized and deserialized. For example, a client may have an older version of a class and the member to which it is connected may have a newer version of the same class.
+
+Portable serialization supports versioning. It is a global versioning, meaning that all portable classes that are serialized through a member get the globally configured portable version.
+
+You can declare Version in the configuration file `hazelcast.xml` using the `portable-version` element, as shown below.
+
+```xml
+<hazelcast>
+    ...
+    <serialization>
+        <portable-version>1</portable-version>
+    </serialization>
+    ...
+</hazelcast>
+```
+
+If you update the class by changing the type of one of the fields or by adding a new field, it is a good idea to upgrade the version of the class, rather than sticking to the global version specified in the `hazelcast.xml` file.
+
+In Python Client, you can achieve this by simply adding the `get_class_version()` method to your class’s implementation of `Portable`, and setting the `CLASS_VERSION` to be different than the default global version.
+
+> Note: If your class that implements `Portable` doesn’t implement this method, its version is set by default to be the global version.
+
+Here is an example implementation of creating a version 2 for the above Foo class:
+
+```python
+from hazelcast.serialization.api import Portable
+
+class Foo(Portable):
+
+    CLASS_ID = 1
+    FACTORY_ID = 1
+    CLASS_VERSION = 2
+    
+    def __init__(self, foo=None, foo2=None):
+        self.foo = foo  
+        self.foo2 = foo2
+        
+    def get_class_id(self):
+        return CLASS_ID
+        
+    def get_factory_id(self):
+        return FACTORY_ID
+        
+    def get_class_version(self):
+        return CLASS_VERSION
+        
+    def write_portable(self, writer):
+        writer.write_utf("foo", self.foo)
+        writer.write_utf("foo2", self.foo2)
+        
+    def read_portable(self, reader):
+        self.foo = reader.read_utf("foo")
+        self.foo2 = reader.read_utf("foo2")
+```
+
+You should consider the following when you perform versioning:
+* It is important to change the version whenever an update is performed in the serialized fields of a class, for example by incrementing the version.
+* If a client performs a Portable deserialization on a field and then that Portable is updated by removing that field on the cluster side, this may lead to a problem.
```

Comment:
I know this is the same content in the Hazelcast IMDG Reference Manual. However, for the sake of improving both documentation, could we give some examples:

"... , this may lead to problems such as ..."